.github/workflows/acr-build-publish-samples.yaml

```diff
@@ -0,0 +1,60 @@
+name: build and push samples projects to acr
+on: 
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: 'Environment to build'
+        required: true
+        default: 'development'
+        type: choice
+        options:
+          - development
+          - staging
+      tag:
+        description: 'Tag to build'
+        required: true
+        default: 'latest'
+        type: string
+permissions:
+  contents: read
+
+jobs:
+  docker:
+    strategy:
+      matrix:
+        include: 
+          - components: stats-api
+            imageName: samples/stats-api
+          - components: frontend
+            imageName: samples/frontend
+          - components: stats-worker    
+            imageName: samples/stats-worker
+          - components: database-api
+            imageName: samples/database-api
+          - components: mi-webapp
+            imageName: samples/mi-webapp
+    runs-on: ubuntu-latest
+    environment: ${{ github.event.inputs.environment }}
+    steps:
+      - name: Harden Runner
+        uses: step-security/harden-runner@eb238b55efaa70779f274895e782ed17c84f2895 # v2.6.1
+        with:
+          egress-policy: audit
+
+      - name: Checkout
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@f95db51fddba0c2d1ec667646a06c2ce06100226 # v3.0.0
+      - name: Login to Docker Hub
+        uses: docker/login-action@343f7c4344506bcbf9b4de18042ae17996df046d # v3.0.0
+        with:
+          registry: ${{ vars.ACR_REGISTRY }}
+          username: ${{ secrets.ACR_USERNAME }}
+          password: ${{ secrets.ACR_PASSWORD }}
```

Comment:
☕️🙏 let’s please avoid this. We ko longer going to work with any secret which is not managed identity. Might be we can discuss this more 1-1 latter.

leaving this as comment here.

Reply:
@Tatsinnit sure sorry I will convert this to draft, still WIP. :)